diff --git a/HDF5config.cmake.orig b/HDF5config.cmake
index 004e600..22adf67 100644
--- a/HDF5config.cmake.orig
+++ b/HDF5config.cmake
@@ -110,7 +110,13 @@ if (NOT DEFINED HPC)
   if (WIN32 AND NOT MINGW)
     set (SITE_OS_NAME "Windows")
     set (SITE_OS_VERSION "WIN10")
-    if (BUILD_GENERATOR STREQUAL "VS201964")
+    if (BUILD_GENERATOR STREQUAL "VS202264")
+      set (CTEST_CMAKE_GENERATOR "Visual Studio 17 2022")
+      set (CMAKE_GENERATOR_ARCHITECTURE "x64")
+      set (SITE_OS_BITS "64")
+      set (SITE_COMPILER_NAME "vs2022")
+      set (SITE_COMPILER_VERSION "17")
+    elseif (BUILD_GENERATOR STREQUAL "VS201964")
       set (CTEST_CMAKE_GENERATOR "Visual Studio 16 2019")
       set (CMAKE_GENERATOR_ARCHITECTURE "x64")
       set (SITE_OS_BITS "64")
