--- a/HDF5options.cmake
+++ b/HDF5options.cmake
@@ -26,7 +26,7 @@
 
 #############################################################################################
 ####      Only build static libraries       ####
-#set (ADD_BUILD_OPTIONS "${ADD_BUILD_OPTIONS} -DBUILD_SHARED_LIBS:BOOL=OFF")
+set (ADD_BUILD_OPTIONS "${ADD_BUILD_OPTIONS} -DBUILD_SHARED_LIBS:BOOL=OFF")
 ####      Only build shared libraries       ####
 #set (ADD_BUILD_OPTIONS "${ADD_BUILD_OPTIONS} -DONLY_SHARED_LIBS:BOOL=OFF")
 ####      Add PIC option on linux/mac       ####
