--- a/Modules/ThirdParty/MetaIO/src/MetaIO/src/metaObject.cxx
+++ b/Modules/ThirdParty/MetaIO/src/MetaIO/src/metaObject.cxx
@@ -1487,7 +1487,7 @@
   mF = MET_GetFieldRecord("CompressedDataSize", &m_Fields);
   if (mF && mF->defined)
   {
-    m_CompressedDataSize = static_cast<long>(mF->value[0]);
+    m_CompressedDataSize = static_cast<std::streamoff>(mF->value[0]);
   }
 
   mF = MET_GetFieldRecord("BinaryData", &m_Fields);
