--- a/loader/linux/icd_linux_envvars.c
+++ b/loader/linux/icd_linux_envvars.c
@@ -25,6 +25,10 @@
 
 #include <stdlib.h>
 
+#if defined(__APPLE__)
+#include <unistd.h>
+#endif
+
 char *khrIcd_getenv(const char *name) {
     // No allocation of memory necessary for Linux.
     return getenv(name);
--- a/test/loader_test/icd_test_match.c
+++ b/test/loader_test/icd_test_match.c
@@ -3,6 +3,9 @@
 #ifndef __APPLE__
 #include <malloc.h>
 #endif
+#ifdef __APPLE__
+#include <stdlib.h>
+#endif
 #include <platform/icd_test_log.h>
 
 int test_icd_match()
