--- a/Rendering/OpenXR/vtkOpenXRManager.cxx
+++ b/Rendering/OpenXR/vtkOpenXRManager.cxx
@@ -42,11 +42,15 @@ vtkOpenXRManager::InstanceVersion vtkOpenXRManager::QueryInstanceVersion(
   // Create the instance with enabled extensions.
   XrInstanceCreateInfo createInfo{ XR_TYPE_INSTANCE_CREATE_INFO };
   createInfo.applicationInfo = XrApplicationInfo{
-    "OpenXR with VTK",      // .applicationName
-    1,                      // .applicationVersion
-    "",                     // .engineName
-    1,                      // .engineVersion
-    XR_CURRENT_API_VERSION, // .apiVersion
+    "OpenXR with VTK",    // .applicationName
+    1,                    // .applicationVersion
+    "",                   // .engineName
+    1,                    // .engineVersion
+#ifdef XR_API_VERSION_1_0 // available with OpenXR 1.1.37 or later:
+    XR_API_VERSION_1_0,   // .apiVersion
+#else                     // for 1.1.36 and earlier:
+    XR_MAKE_VERSION(1, 0, XR_VERSION_PATCH(XR_CURRENT_API_VERSION)), // .apiVersion
+#endif
   };
   createInfo.enabledExtensionCount = static_cast<uint32_t>(enabledExtensions.size());
   createInfo.enabledExtensionNames = enabledExtensions.data();
@@ -804,11 +808,15 @@ bool vtkOpenXRManager::CreateInstance()
   createInfo.enabledExtensionNames = enabledExtensions.data();
 
   XrApplicationInfo applicationInfo = {
-    "OpenXR with VTK",      // .applicationName
-    1,                      // .applicationVersion
-    "",                     // .engineName
-    1,                      // .engineVersion
-    XR_CURRENT_API_VERSION, // .apiVersion
+    "OpenXR with VTK",    // .applicationName
+    1,                    // .applicationVersion
+    "",                   // .engineName
+    1,                    // .engineVersion
+#ifdef XR_API_VERSION_1_0 // available with OpenXR 1.1.37 or later:
+    XR_API_VERSION_1_0,   // .apiVersion
+#else                     // for 1.1.36 and earlier:
+    XR_MAKE_VERSION(1, 0, XR_VERSION_PATCH(XR_CURRENT_API_VERSION)), // .apiVersion
+#endif
   };
 
   createInfo.applicationInfo = applicationInfo;
