--- a/Rendering/OpenXR/vtkOpenXRManager.cxx
+++ b/Rendering/OpenXR/vtkOpenXRManager.cxx
@@ -717,7 +717,11 @@
     1,                      // .applicationVersion
     "",                     // .engineName
     1,                      // .engineVersion
-    XR_CURRENT_API_VERSION, // .apiVersion
+#ifdef XR_API_VERSION_1_0   // available with OpenXR 1.1.37 or later:
+    XR_API_VERSION_1_0,     // .apiVersion
+#else                       // for 1.1.36 and earlier:
+    XR_MAKE_VERSION(1, 0, XR_VERSION_PATCH(XR_CURRENT_API_VERSION)), // .apiVersion
+#endif
   };
 
   createInfo.applicationInfo = applicationInfo;
