--- a/Rendering/OpenXR/vtkOpenXRCamera.cxx
+++ b/Rendering/OpenXR/vtkOpenXRCamera.cxx
@@ -115,13 +115,6 @@ void vtkOpenXRCamera::Render(vtkRenderer* ren)
   int renSize[2];
   win->GetRenderBufferSize(renSize[0], renSize[1]);
 
-  // update mats
-  vtkMatrix4x4* wcvc;
-  vtkMatrix3x3* normMat;
-  vtkMatrix4x4* vcdc;
-  vtkMatrix4x4* wcdc;
-  this->GetKeyMatrices(ren, wcvc, normMat, vcdc, wcdc);
-
   // if were on a stereo renderer draw to special parts of screen
   if (this->LeftEye)
   {
--- a/Rendering/VR/vtkVRHMDCamera.cxx
+++ b/Rendering/VR/vtkVRHMDCamera.cxx
@@ -115,6 +115,9 @@ void vtkVRHMDCamera::GetKeyMatrices(vtkRenderer* ren, vtkMatrix4x4*& wcvc, vtkMa
 
       vtkMatrix4x4::Multiply4x4(this->LeftEyeToProjectionMatrix, this->PhysicalToLeftEyeMatrix,
         this->PhysicalToProjectionMatrixForLeftEye);
+
+      this->VCDCMatrix->DeepCopy(this->LeftEyeToProjectionMatrix);
+      this->VCDCMatrix->Transpose();
     }
     else
     {
@@ -129,6 +132,9 @@ void vtkVRHMDCamera::GetKeyMatrices(vtkRenderer* ren, vtkMatrix4x4*& wcvc, vtkMa
 
       vtkMatrix4x4::Multiply4x4(this->RightEyeToProjectionMatrix, this->PhysicalToRightEyeMatrix,
         this->PhysicalToProjectionMatrixForRightEye);
+
+      this->VCDCMatrix->DeepCopy(this->RightEyeToProjectionMatrix);
+      this->VCDCMatrix->Transpose();
     }
 
     this->KeyMatrixTime.Modified();
@@ -138,15 +144,7 @@ void vtkVRHMDCamera::GetKeyMatrices(vtkRenderer* ren, vtkMatrix4x4*& wcvc, vtkMa
   wcdc = this->WCDCMatrix;
   wcvc = this->WCVCMatrix;
   normMat = this->NormalMatrix;
-
-  if (this->LeftEye)
-  {
-    vcdc = this->LeftEyeToProjectionMatrix;
-  }
-  else
-  {
-    vcdc = this->RightEyeToProjectionMatrix;
-  }
+  vcdc = this->VCDCMatrix;
 }
 
 //------------------------------------------------------------------------------
