--- a/Modules/Bridge/VtkGlue/include/vtkCaptureScreen.h
+++ b/Modules/Bridge/VtkGlue/include/vtkCaptureScreen.h
@@ -55,7 +55,7 @@
     vtkCaptureScreen ( const vtkCaptureScreen& );
     void operator = ( const vtkCaptureScreen& );
 
-    vtkRenderWindow* m_Renderer;
+    mutable vtkRenderWindow* m_Renderer;
 
     void Capture( vtkRenderWindow* iRenderer,
                   const std::string& iFileName ) const
