--- Modules/Bridge/VtkGlue/include/vtkCaptureScreen.h.old	2020-06-17 09:38:17.473860162 +0200
+++ Modules/Bridge/VtkGlue/include/vtkCaptureScreen.h	2020-06-17 09:33:05.970728723 +0200
@@ -55,7 +55,7 @@
     vtkCaptureScreen ( const vtkCaptureScreen& );
     void operator = ( const vtkCaptureScreen& );
 
-    vtkRenderWindow* m_Renderer;
+    mutable vtkRenderWindow* m_Renderer;
 
     void Capture( vtkRenderWindow* iRenderer,
                   const std::string& iFileName ) const
