--- CMakeLists.txt.old	2021-05-28 15:17:11.000000000 +0200
+++ CMakeLists.txt	2021-07-05 09:03:55.470726300 +0200
@@ -29,7 +29,7 @@
     cmake_policy(SET ${pnew} NEW)
   endif()
 endforeach()
-foreach(pold "CMP0120") # CMP0120 will require non-trivial effort to address
+foreach(pold "CMP0120" "CMP0091") # CMP0120 will require non-trivial effort to address
   if(POLICY ${pold})
     cmake_policy(SET ${pold} OLD)
   endif()
