--- a/Modules/Segmentation/RegionGrowing/include/itkConfidenceConnectedImageFilter.hxx
+++ b/Modules/Segmentation/RegionGrowing/include/itkConfidenceConnectedImageFilter.hxx
@@ -233,7 +233,7 @@
   upper = m_Mean + m_Multiplier * std::sqrt(m_Variance);
 
   // Find the highest and lowest seed intensity.
-  InputRealType lowestSeedIntensity = itk::NumericTraits<InputImagePixelType>::max();
+  InputRealType lowestSeedIntensity = static_cast<InputRealType>(itk::NumericTraits<InputImagePixelType>::max());
   InputRealType highestSeedIntensity = itk::NumericTraits<InputImagePixelType>::NonpositiveMin();
   typename SeedsContainerType::const_iterator si = m_Seeds.begin();
   typename SeedsContainerType::const_iterator li = m_Seeds.end();
