--- a/Modules/Numerics/Statistics/include/itkMaskedImageToHistogramFilter.hxx
+++ b/Modules/Numerics/Statistics/include/itkMaskedImageToHistogramFilter.hxx
@@ -50,7 +50,7 @@
   maskIt.GoToBegin();
   HistogramMeasurementVectorType m(nbOfComponents);
 
-  min.Fill(NumericTraits<ValueType>::max());
+  min.Fill(static_cast<ValueRealType>(NumericTraits<ValueType>::max()));
   max.Fill(NumericTraits<ValueType>::NonpositiveMin());
   while (!inputIt.IsAtEnd())
   {
