--- a/Modules/Filtering/MathematicalMorphology/include/itkReconstructionImageFilter.hxx
+++ b/Modules/Filtering/MathematicalMorphology/include/itkReconstructionImageFilter.hxx
@@ -96,12 +96,12 @@
   MaskImageConstPointer   maskImageP;
 
   ISizeType padSize;
+  padSize.Fill(1);
 
   if (m_UseInternalCopy)
   {
     auto MaskPad = PadType::New();
     auto MarkerPad = PadType::New();
-    padSize.Fill(1);
 
     MaskPad->SetConstant(m_MarkerValue);
     MarkerPad->SetConstant(m_MarkerValue);
