--- a/Modules/Remote/RTK/include/rtkCudaFFTProjectionsConvolutionImageFilter.h
+++ b/Modules/Remote/RTK/include/rtkCudaFFTProjectionsConvolutionImageFilter.h
@@ -72,11 +72,11 @@
   CudaFFTProjectionsConvolutionImageFilter();
   ~CudaFFTProjectionsConvolutionImageFilter() {}
 
-  virtual void
-  GPUGenerateData();
+  void
+  GPUGenerateData() override;
 
-  virtual FFTInputImagePointer
-  PadInputImageRegion(const RegionType & inputRegion);
+  FFTInputImagePointer
+  PadInputImageRegion(const RegionType & inputRegion) override;
 
 private:
   CudaFFTOutputImagePointer m_KernelFFTCUDA;
