--- /tools/rtkCudaForwardProjectionImageFilter.h	2020-08-25 10:01:32.505158622 +0200
+++ /tools/sb/itk/src-5.1.0/Modules/Remote/RTK/include/rtkCudaForwardProjectionImageFilter.h	2020-08-25 09:59:10.151602693 +0200
@@ -78,8 +78,8 @@
   CudaForwardProjectionImageFilter();
   ~CudaForwardProjectionImageFilter(){};
 
-  virtual void
-  GPUGenerateData();
+  void
+  GPUGenerateData() override;
 
 private:
   double m_StepSize;
