--- /tools/rtkCudaBackProjectionImageFilter.h	2020-08-25 10:01:51.113472734 +0200
+++ /tools/sb/itk/src-5.1.0/Modules/Remote/RTK/include/rtkCudaBackProjectionImageFilter.h	2020-08-25 10:00:02.168486130 +0200
@@ -74,8 +74,8 @@
   CudaBackProjectionImageFilter();
   virtual ~CudaBackProjectionImageFilter(){};
 
-  virtual void
-  GPUGenerateData();
+  void
+  GPUGenerateData() override;
 };
 
 } // end namespace rtk
