--- /tools/rtkCudaFDKConeBeamReconstructionFilter.h	2020-08-25 10:11:58.269585939 +0200
+++ /tools/sb/itk/src-5.1.0/Modules/Remote/RTK/include/rtkCudaFDKConeBeamReconstructionFilter.h	2020-08-25 10:12:09.389649761 +0200
@@ -76,8 +76,8 @@
   CudaFDKConeBeamReconstructionFilter();
   ~CudaFDKConeBeamReconstructionFilter() {}
 
-  virtual void
-  GPUGenerateData();
+  void
+  GPUGenerateData() override;
 
 }; // end of class
 
